web/.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
```

Comment:
One argument for .eslintrc.js over .eslint.json  ---- You can comment in .js

Reply:
Apparently you can also add comments in .eslint.json [https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-file-formats](url)